catalystwan/response.py

```diff
@@ -104,6 +105,14 @@ def response_history_debug(response: Optional[Response], request: Union[Request,
     return "\n".join(response_debugs)
 
 
+def auth_response_debug(response: Response, title: str = "Auth") -> str:
+    if environ.get("catalystwan_auth_trace") is not None:
```

Comment:
maybe upper case latters for catalystwan_auth_trace and add information to readme?

Reply:
we already use `catalystwan_devel`, `catalystwan_export_endpoints`
Will update CONTRIBUTING.md